repository-data/application/src/main/resources/hcm-config/namespaces/hee/linkBlock.yaml

```diff
@@ -0,0 +1,71 @@
+definitions:
+  config:
+    /hippo:namespaces/hee/linkBlock:
+      jcr:primaryType: hipposysedit:templatetype
+      jcr:mixinTypes: ['editor:editable', 'mix:referenceable']
+      jcr:uuid: 440b9937-3b87-42ed-8b2f-11bfb331f1d3
+      /hipposysedit:nodetype:
+        jcr:primaryType: hippo:handle
+        jcr:mixinTypes: ['mix:referenceable']
+        jcr:uuid: 3ac705f0-9483-4c2a-8583-a3c2921bdcf6
+        /hipposysedit:nodetype:
+          jcr:primaryType: hipposysedit:nodetype
+          jcr:mixinTypes: ['mix:referenceable', 'hipposysedit:remodel']
+          jcr:uuid: 4866b571-c0e9-4e50-a6a2-8008a8428e4e
+          hipposysedit:node: true
+          hipposysedit:supertype: ['hippo:compound', 'hippostd:relaxed']
+          hipposysedit:uri: http://www.heecmsplatform.com/hee/nt/1.0
+          /blockTitle:
```

Comment:
Could we change `blockTitle` as a `string` type field with `TextBox` editor (in order to be inline with other cards/blocks) ?

Reply:
Yes. Commited on 548764f